src/components/participants-list/participants-list.tsx

```diff
@@ -0,0 +1,43 @@
+import { Participant } from "@/app/events/[event]/participants/page";
+
+interface ParticipantListProps {
+  participants: Participant[];
+}
+
+export function ParticipantsList({ participants }: ParticipantListProps) {
+  return (
+    participants.length && (
```

Comment:
This logic shouldn't be here. You can move it to the place where we render this component.

Reply:
Done